.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,42 @@
+# iTwin.js CodeQL Build
+
+name: "CodeQL"
+
+on:
```

Comment:
could you also add workflow dispatch, so we can manually run this pipeline

Reply:
added